### DISCUSSION TITLE
ActiveMQ support?

### DISCUSSION THREAD

**Discussion by japj** [microsoft/aspire#6694] [Q&A] [answered]
Hi,

I am completely new to Aspire and am looking into using it in a project that is currently using ActiveMQ.
I saw there is an ActiveMQ PR #4094, but no progress after it was created in May. 
Is this something that should be in the CommunityToolkit instead?

**Comment by davidfowl** [comment, marked as answer]
> Is this something that should be in the CommunityToolkit instead?

yes.

**Comment by anoordover** [comment]
See https://github.com/CommunityToolkit/Aspire/pull/273. Moved to CommunityToolkit.
